src/gluonts/dataset/pandas.py

```diff
@@ -207,23 +204,24 @@ def __len__(self) -> int:
         return len(self._data_entries)
 
     def __str__(self) -> str:
-        return (
-            f"PandasDataset<"
-            f"size={len(self)}, "
-            f"freq={self.freq}, "
-            f"num_dynamic_real={self.num_feat_dynamic_real}, "
-            f"num_past_dynamic_real={self.num_past_feat_dynamic_real}, "
-            f"num_static_real={self.num_feat_static_real}, "
-            f"num_static_cat={self.num_feat_static_cat}, "
-            f"cardinalities={self.cardinalities}>"
```

Comment:
Why not printing the cardinalities? I think it can be useful for quick validation